index.php

```diff
@@ -1,80 +1,58 @@
         <meta name=’viewport” content=”width=device-width, initial-scale=0.1”>
     </head>
 
     <body>
-        <h1>AFL Player Cards</h1>
-
-        <section class="container">
-            <div class="card1">
-                <div class="card_top">
-                    <h2 class="player_name"></h2>
-                    <h2 class="club"></h2>
-                </div>
-                <img>
-                <div class="stats">
-                    <p class="position"></p>
-                    <p class="DOB"></p>
-                    <p class="height"></p>
-                    <p class="weight"></p>
-                </div>
-            </div>
-
-            <div class="card2">
-                <div class="card_top">
-                    <h2 class="player_name"></h2>
-                    <h2 class="club"></h2>
-                </div>
-                <img>
-                <div class="stats">
-                    <p class="position"></p>
-                    <p class="DOB"></p>
-                    <p class="height"></p>
-                    <p class="weight"></p>
-                </div>
-            </div>
-
-            <div class="card3">
-                <div class="card_top">
-                    <h2 class="player_name"></h2>
-                    <h2 class="club"></h2>
-                </div>
-                <img>
-                <div class="stats">
-                    <p class="position"></p>
-                    <p class="DOB"></p>
-                    <p class="height"></p>
-                    <p class="weight"></p>
-                </div>
-            </div>
-
-            <!-- etc -->
+        <h1>AFL Player Cards Collection</h1>
+        <p class="intro">Welcome to my awesome collection of Australian Rules Football player cards. Each card displays the photo and stats of the some of the best players in the league.</p>
+
+        <section class="grid_container">
+
+            <?php
+
+            foreach($results as $player) {
+                echo '<div class="card ' . str_replace(' ', '_',$player['club']) . '">';
+                echo '<div class="card_top">';
+                echo '<h2 class="player_name">' . $player['player_first_name'] . ' ' . $player['player_last_name'] . '</h2>';
+                echo '<h3 class="club">' . $player['club'] . ' </h3>';
+                echo '</div>';
+                echo '<img src="' . $player['image'] . '">';
+                echo '<div class="stats">';
+                echo '<p class="position">' . 'Position: ' . $player['position'] . ' </p>';
+                echo '<p class="DOB">' . 'DOB: ' . $player['DOB'] . ' </p>';
+                echo '<p class="height">' . 'Height: ' . $player['height'] . ' m' . ' </p>';
+                echo '<p class="weight">' . 'Weight: ' . $player['weight'] . ' kg' . ' </p>';
+                echo '</div>';
+                echo '</div>';
```

Comment:
I'd put this into a function, so you'd have like:
foreach($results as $player) {
    showPlayerCard($player);
}

And then the function would do the echo part, just makes it a bit nicer and becomes unit testable :)